storage/local/interface.go

```diff
@@ -47,6 +47,9 @@ type Storage interface {
 	// The iterator will never return samples older than retention time,
 	// relative to the time NewIterator was called.
 	NewIterator(clientmodel.Fingerprint) SeriesIterator
+	// Drop all time series associated with the given fingerprints. This operation
+	// will not show up in the series operations metrics.
```

Comment:
The "purge" operation basically signals "a time series has not received any updated for the whole retention period and has therefore been deleted". If you manually delete time series, you are pretty aware that you did that. Showing those events in the same metric as the "real" purges is probably more confusing than helpful.